pinkopy/commvault.py

```diff
@@ -19,9 +15,10 @@ class CommvaultSession(BaseSession):
     the past.
     """
     def __init__(self, *args, **kwargs):
-        self.clients = ClientSession(*args, **kwargs)
-        self.subclients = SubclientSession(*args, **kwargs)
-        self.jobs = JobSession(*args, **kwargs)
+        cache_methods = None
+        self.clients = ClientSession(cache_methods, *args, **kwargs)
+        self.subclients = SubclientSession(cache_methods, *args, **kwargs)
+        self.jobs = JobSession(cache_methods, *args, **kwargs)
```

Comment:
I don't know what drove this to be added here. It would not break the subsessions' ability to cache using the defaults, but it would overwrite the argument passed in, so it breaks custom cache methods. This may have been to resolve a test, but it doesn't pass muster for functionality. The way it is coded since your previous base may already resolve this issue. It changed a little.

We are now instantiating the super class in `CommvaultSession`, so cache_methods is set here, though it isn't used here. It allows us to report correct information with those properties.